DEVELOPING.md

```diff
@@ -207,3 +213,35 @@ Please refer to the MAINTAINERS file for maintenance guidelines and contact info
 ## Contact and Support
 
 If you need help or have any questions, please open an issue on GitHub or contact us at support@neuralmagic.com.
+
+This will run automatic code styling using `black` and `isort` and test that the
```

Comment:
This text is out of place here, was this a miscopy?

Reply:
Ohh. Yes probably